doc/classes/ImmediateGeometry.xml

```diff
@@ -50,14 +51,14 @@
 			<return type="void">
 			</return>
 			<description>
-				Clear everything that was drawn using begin/end.
+				Clears everything that was drawn using begin/end.
 			</description>
 		</method>
 		<method name="end">
 			<return type="void">
 			</return>
 			<description>
-				Call this when done adding a batch of geometry, otherwise it can't be displayed.
+				Calls this when done adding a batch of geometry, otherwise it can't be displayed.
```

Comment:
I don't think this description is right. You're supposed to use this function when you are finished with a batch.

   Ends a drawing context and displays the results.